Marlin/src/module/stepper_indirection.h

```diff
@@ -49,7 +49,11 @@
 // TMC26X drivers have STEP/DIR on normal pins, but ENABLE via SPI
 #if ENABLED(HAVE_TMCDRIVER)
   #include <SPI.h>
-  #include <TMC26XStepper.h>
+  #if defined(STM32F7)
```

Comment:
Anything that breaks HAL encapsulation like this needs to be only as a last resort

Reply:
yup, i understand, do you have a better way of doing it ?